compiler/rustc_middle/src/thir.rs

```diff
@@ -783,16 +783,13 @@ pub enum PatKind<'tcx> {
     },
 
     /// One of the following:
-    /// * `&str` (represented as a valtree), which will be handled as a string pattern and thus
+    /// * `&str`/`&[u8]` (represented as a valtree), which will be handled as a string pattern and thus
```

Comment:
```suggestion
    /// * `&str`/`&[u8]` (represented as a valtree), which will be handled as a string/slice pattern and thus
```

I assume (I'm not particularly well-versed in `rustc_pattern_analysis`)